Weave basics docs revamp DOCS-1896 (#2193)
wandb/docs: weave/guides/tracking/update-call.mdx

---
title: "Update and delete Calls"
description: "Modify display names, add feedback, and delete Calls in W&B Weave"
---

> Calls are mostly immutable after creation. A few mutations are supported:

Reviewer: ```suggestion
Most Call properties are immutable after creation. The following mutations are supported:
```
